tests/test_workers/test_pkg_managers/test_pip.py

```diff
@@ -2548,23 +2561,33 @@ def test_download_vcs_package(
                 },
             ),
             (
+                # Port should (?) be part of host
```

Comment:
Yes or no? This would mean we save the file in `deps/pip/github.com:443/namespace/repo/repo-external-gitcommit-...` (or `deps/pip/my-private-git-server:8083/foo/bar/...`)

Reply:
This should be important information for locating a resource. Are we storing ports for npm?

Reply:
I'm going to keep the port number, like you said it may be important info for locating a resource